server/app/user_handler.go

```diff
@@ -655,3 +655,20 @@ func (a *App) ActivateVoucherHandler(req *http.Request) (interface{}, Response)
 		Data:    nil,
 	}, Ok()
 }
+
+// func (a *App) GetMaintenanceHandler(req *http.Request) (interface{}, Response) {
```

Comment:
not sure why it is commented?

Reply:
> not sure why it is commented?

it was before I made nextlaunch, I thought I was supposed to edit in maintenance, I commented however forgot to totally remove, which I will now